### DISCUSSION TITLE
A module dissmissed

### DISCUSSION THREAD

**Discussion by lclty** [mikumifa/biliTickerBuy#141] [提问专区]
![image](https://github.com/mikumifa/biliTickerBuy/assets/102938905/b86cfa8e-bfdd-44f7-ab16-ab5ab8b7d764)

- Git clone, use the commit [2119184](https://github.com/mikumifa/biliTickerBuy/commit/2119184459b16433694d39d784a92a6d584a0592) , I'm sure I have been installed all of the requirements . However , while I type `python main.py` , there is an module error output(picture 1). I checked the code ./tab/go.py , I am not find the function "bili_ticket_gt_python" 's def.

**Comment by lclty** [comment]
I think this is likely due to you placing the definition of this function in another module. I suggest you modify it. I don't want to use the exe program compiled by PyInstaller, and I also need to run it on Ubuntu (after obtaining the cookie, it seems like I don't need WebDriver anymore, so I should be able to run it on Ubuntu).

**Comment by Amorter** [comment]
应该是可以的，一次上传的时候不小心把模块导入写错了，稍等pr

**Reply by Amorter** [reply]
修改go.py中的from 模块为  util.dynimport即可
